Form/DataTransformer/ModelsToArrayTransformer.php

```diff
@@ -39,20 +39,46 @@ class ModelsToArrayTransformer implements DataTransformerInterface
 
     /**
      * @var ModelChoiceList
+     * @deprecated
      */
     protected $choiceList;
 
     /**
      * ModelsToArrayTransformer constructor.
      *
      * @param ModelChoiceList|LazyChoiceList|ModelChoiceLoader $choiceList
-     * @param ModelManagerInterface                            $modelManager
+     * @param $modelManager
      * @param $class
      *
      * @throws RuntimeException
      */
-    public function __construct($choiceList, ModelManagerInterface $modelManager, $class)
+    public function __construct($choiceList, $modelManager, $class)
```

Comment:
`$class` should be optional now, right? So `$class = null`. Please add a test to show that this class can be instanciated with 2 arguments just fine.